src/api/azdevops-api.ts

```diff
@@ -692,4 +691,26 @@ export class WorkItemTreeItem extends vscode.TreeItem {
             return remoteRefs;
         }
     }
+
+    private getBranchNameProposal() {
+        const gitPrefix = vscode.workspace.getConfiguration('git').get('branchPrefix', "");
+        let branchNameProposal = gitPrefix !== "" ? `${gitPrefix}` : "";
+        let safeDescription = this.label.trim().toLowerCase().replace(/ /g, "-");
```

Comment:
should we also guard against special characters? I would probably just take letters and numbers and replace everything else with a dash. If that creates a weird name, the user can always change the proposal

Reply:
Isn't that approximately what I'm doing? I'm replacing first spaces with a dash and then remove everything that's not a word character or a dash.🤔

Reply:
absolutely. Sorry, I only saw the first replace and failed to note the second one